commands/volumes/upload.go

```diff
@@ -56,28 +58,64 @@ func handleUploadVolumeCmd(c client.Client) error {
 		params["Org"] = c.Org
 	}
 
+	var file *os.File
```

Comment:
You're not closing "file" and will leak file descriptors.

Reply:
Fixed.